readme.md

```diff
@@ -1,139 +1,99 @@
-# RavenDB - An ACID NoSQL Document Database
-This repository contains source code for the [RavenDB](https://ravendb.net/) document database. With a RavenDB database you can set up a NoSQL data architecture or add a NoSQL layer to your current relational database.
+<h1 align="center"><img src="docs/Server guide banner.png" alt="Ory Keto - Open Source & Cloud Native Access Control Server">RavenDB - Modern ACID NoSQL Database Solution</h1>
```

Comment:
- alt text is carried over from AI prompt?
- What would "solution" mean in this context?

Reply:
done, removed the "solution"